modules/client-common/CHANGELOG.md

```diff
@@ -21,6 +21,12 @@ TEMPLATE:
 
 ### Added
 
+- Added `UPGRADE_PLUGIN_PERMISSION` to `PermissionType` enum
```

Comment:
More things happened in this PR @josemarinas

Reply:
Not on this package